debian/rules.in

```diff
@@ -20,6 +20,7 @@ XPMEM_OPT = $(shell echo ${DEB_BUILD_PROFILES} | grep -qw xpmem \
 
 override_dh_auto_configure:
 	./contrib/configure-release-mt --prefix=/usr \
+		--with-rdmacm=yes --with-verbs=yes --with-mlx5=yes --enable-cma \
```

Comment:
Do we really need "=yes" or can be just --with-rdmacm --with-verbs ...

Reply:
fixed (confirmed implicit means `yes`)